src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/FileUploader.vue

```diff
@@ -64,16 +87,53 @@ export default defineComponent({
       selectedFiles: [],
       showErrorBanner: false,
       errorBannerMessage: "",
+      firstLoad: true,
     };
   },
+  computed: {
+    fileErrors() {
```

Comment:
How are the errors displayed if put into rules on the input component? I assume vuetify just puts the error underneath the input? We created the `fileErrors` list and associate it with a `FileItem` for individual file errors... Maybe we need to discuss how to handle the more general errors. A few initial thoughts I have:

- We may not need `filesInProgress`, do we have a requirement that we can't upload multiple 'batches' simultaneously? If not then we can safely remove this as I believe we are handling multiple 'batches' of uploads concurrently now. 
- I thought we removed the need `allFilesTooLarge` requirement, I'd much rather we go with a per file size limit and then limit the number of files. We may need to talk to design on this if it's not clear in Figma...

Reply:
the filesInProgress is to stop the user from submitting while a file is still being uploaded. 

The user can still upload multiple batches. Like 1 file and then upload another file while the previous is still going on. 

previously we emitted the list of files to the parent component to determine if files were still being uploaded to prevent the user from submitting 
Example replyToMessage.vue https://github.com/bcgov/ECC-ECER/blob/eecbba446a60fdb1150ebff2a8148c21a43f7c28/src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ReplyToMessage.vue#L111

I wanted to improve this and not require the parent component to calculate the files in progress stuff. 

So I did compute methods + a hidden input (like we did in workReferences.vue. To stop the user from progressing in the form.